README.md

```diff
@@ -8,6 +8,20 @@ A number of these crawlers have been sourced from [Dark Visitors](https://darkvi
 
 If you'd like to add information about a crawler to the list, please make a pull request with the bot name added to `robots.txt`, `ai.txt`, and any relevant details in `table-of-bot-metrics.md` to help people understand what's crawling.
 
+## Usage
+
+Many visitors will find these files from this repository most useful:
```

Comment:
Who are the "visitors"? I suggest adopting a more formal tone and breaking out subsections for `robots.txt` and `.htaccess`.